bundles/org.openhab.io.homekit/README.md

```diff
@@ -994,3 +994,47 @@ You can verify this with [Discovery DNS iOS app](https://apps.apple.com/us/app/d
   - if sf is equal 1, openHAB is accepting pairing from new iOS device. 
   - if sf is equal 0 (as on screenshot), openHAB is already paired and does not accept any new pairing request. you can reset pairing using `openhab:homekit clearPairings` command in karaf console.
 - if you see openHAB bridge and sf is equal 1 but you dont see openHAB in home app, probably you home app still think it is already paired with openHAB. remove your home from home app and restart iOS device.
+
+### Re-adding the openHAB HomeKit bridge reports that a bridge is already added, even though it has clearly been removed.
+
+There are various reasons this may happen.
+Try the following:
+
+* In [openhab-cli](https://www.openhab.org/docs/administration/console.html), run `openhab:homekit clearPairings`.
+On openHAB 3.4.0 and earlier, reboot openHAB.
```

Comment:
I'm not sure of the policy on referring to earlier openHAB versions in current docs. Are you wanting this cherry-picked to the 3.4 branch? @openhab/add-ons-maintainers

Reply:
I'll just remove it for now.